Userland/Libraries/LibJS/Parser.cpp

```diff
@@ -2462,6 +2471,7 @@ NonnullRefPtr<ReturnStatement const> Parser::parse_return_statement()
         syntax_error("'return' not allowed outside of a function");
 
     consume(TokenType::Return);
+    maybe_force_slash_as_regex();
```

Comment:
Adding `maybe_force_slash_as_regex()` everywhere the token is used "normally" seems like something thats easy to forget, so maybe we could add the reverse check if those tokens are used as identifiers?

Reply:
Perhaps consume could have an extra parameter which automatically does maybe_force_slash_as_regex for the proper tokens by default?